README.md

```diff
@@ -19,8 +19,9 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 
 ## Templates
 - [tauri-svelte-template](https://github.com/probablykasper/tauri-svelte-template) - Svelte template with cross-platform GitHub action builds, Vite, TypeScript, Svelte Preprocess, hot module replacement, ESLint and Prettier.
-- [tauri-react-template](https://github.com/oSethoum/tauri-react-template) - React template with Vite, TypeScript, hot module replacement.
+- [tauri-react-template](https://github.com/oSethoum/tauri-react-template) - React template with Vite, TypeScript, hot module replacement. 
```

Comment:
Can you get rid of this extra space here?

Reply:
Sure thing 👍